File: CS/DxSample/Grid/GridControlEx.cs
using DevExpress.XtraGrid;
using DevExpress.XtraGrid.Registrator;

namespace DxSample.Grid {
    public class GridControlEx :GridControl {
        protected override void RegisterAvailableViewsCore(InfoCollection collection) {
            base.RegisterAvailableViewsCore(collection);
            collection.Add(new GridViewExInfoRegistrator());
        }
    }
}

File: VB/DxSample/Grid/GridControlEx.vb
Imports DevExpress.XtraGrid
Imports DevExpress.XtraGrid.Registrator

Namespace DxSample.Grid

    Public Class GridControlEx
        Inherits GridControl

        Protected Overrides Sub RegisterAvailableViewsCore(ByVal collection As InfoCollection)
            MyBase.RegisterAvailableViewsCore(collection)
            collection.Add(New GridViewExInfoRegistrator())
        End Sub
    End Class
End Namespace
